zeebe/scheduler/src/main/java/io/camunda/zeebe/scheduler/future/CompletableActorFuture.java

```diff
@@ -312,6 +313,19 @@ public <U> ActorFuture<U> andThen(
     return nextFuture;
   }
 
+  @Override
+  public <U> ActorFuture<U> andThen(
+      final BiFunction<V, Throwable, ActorFuture<U>> next, final Executor executor) {
+    final ActorFuture<U> nextFuture = new CompletableActorFuture<>();
+    onComplete(
+        (thisResult, thisError) -> {
+          final var future = next.apply(null, thisError);
```

Comment:
:x: Unless I'm missing something, we should call it with `thisResult` **and** `thisError`

Reply:
Good indicator for missing tests too